showCrime/showCrime/settings.py

```diff
@@ -1,3 +1,12 @@
+"""
+django app for https://OakCrime.org
+"""
+
+__author__ = "rik@electronicArtifacts.com"
+__credits__ = ["clinton.blackburn@gmail.com","actionspeakslouder@gmail.com"]
+__date__ = "190128"
+__version__ = "0.1.0-alpha"
```

Comment:
Versioning is a nebulous concept for a web application that is potentially continuously deployed. If you're using [semantic versioning](https://semver.org/), what counts as a major versus a minor version? Regardless, what is the motivation for placing the version identifier in this file as opposed to some file at the root of the repo?